### ISSUE TITLE
Missing locales

### ISSUE THREAD

**Issue by sabaini** [popey/matterhorn-snap#9]
I've setup the example activityNotifyCommand https://github.com/matterhorn-chat/matterhorn/blob/master/notification-scripts/notify 

This sends notifications ok, although matterhorn prints an error message when triggered. The reason I believe is b/c of missing locales. The debug log has:

```
...
Stderr:
bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
```

While my base system has that locale installed it seems the snap is missing locales:

```
sudo snap run --shell matterhorn -c 'echo hello'
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
hello
```

I can work around this by exporting LC_ALL=C.UTF-8 but I'm wondering if it would be possible to get proper locale support?

**Comment by 3v1n0** [comment]
Yes, this should also be the root cause for https://github.com/matterhorn-chat/matterhorn/issues/623
